website/webpack.config.js

```diff
@@ -114,7 +115,11 @@ const config = () => {
       ],
     },
 
-    plugins: [],
+    plugins: [
```

Comment:
Same applies here. I think the  CI/CD targets deployments and since we will being using served static fields. No need for html index page.